DesktopUI2/DesktopUI2/ViewModels/CollaboratorsViewModel.cs

```diff
@@ -134,189 +134,243 @@ private async void SearchUsers()
   [DependsOn(nameof(AddedUsers))]
   private bool CanSaveCommand(object parameter)
   {
-    foreach (var user in AddedUsers)
+    try
     {
-      if (Utils.IsValidEmail(user.Name) && !_stream.Stream.pendingCollaborators.Any(x => x.title == user.Name))
-        return true;
-      if (
-        !_stream.Stream.collaborators.Any(x => x.id == user.Id)
-        && !_stream.Stream.pendingCollaborators.Any(x => x.id == user.Id)
-      )
-        return true;
-      if (
-        !_stream.Stream.collaborators.Any(x => x.id == user.Id && x.role == user.Role)
-        && !_stream.Stream.pendingCollaborators.Any(x => x.id == user.Id && x.role == user.Role)
-      )
-        return true;
+      foreach (var user in AddedUsers)
+      {
+        if (Utils.IsValidEmail(user.Name) && !_stream.Stream.pendingCollaborators.Any(x => x.title == user.Name))
+          return true;
+        if (
+          !_stream.Stream.collaborators.Any(x => x.id == user.Id) &&
+          !_stream.Stream.pendingCollaborators.Any(x => x.id == user.Id)
+        )
+          return true;
+        if (
+          !_stream.Stream.collaborators.Any(x => x.id == user.Id && x.role == user.Role) &&
+          !_stream.Stream.pendingCollaborators.Any(x => x.id == user.Id && x.role == user.Role)
+        )
+          return true;
+      }
+
+      foreach (var user in _stream.Stream.collaborators)
+        if (!AddedUsers.Any(x => x.Id == user.id))
+          return true;
+      foreach (var user in _stream.Stream.pendingCollaborators)
+        if (!AddedUsers.Any(x => x.Id == user.id))
+          return true;
     }
-    foreach (var user in _stream.Stream.collaborators)
-      if (!AddedUsers.Any(x => x.Id == user.id))
-        return true;
-    foreach (var user in _stream.Stream.pendingCollaborators)
-      if (!AddedUsers.Any(x => x.Id == user.id))
-        return true;
+    catch (Exception ex)
+    {
+      SpeckleLog.Logger.Warning(
+        ex,
+        "Swallowing exception in {methodName}: {exceptionMessage}",
+        nameof(CanSaveCommand),
+        ex.Message
+      );
+    }
```

Comment:
Shouldn't this be a `Fatal` log, since it would indicate something unexpected went wrong that prevented us from fulfilling the request.